File: JAVA/PetShop.java
class PetShop
{
  private int id;
  private String nama;
  private int harga;
  private int stok;

  // konstruktor
  public PetShop(){

  }

  public PetShop(int id, String nama, int harga, int stok){
    this.id = id;
    this.nama = nama;
    this.harga = harga;
    this.stok = stok;
  }

  // getter & setter
  public int getId(){
    return id;
  }

  public void setId(int id){
    this.id = id;
  }

  public String getNama(){
    return nama;
  }

  public void setNama(String nama){
    this.nama = nama;
  }

  public int getHarga(){
    return harga;
  }

  public void setHarga(int harga){
    this.harga = harga;
  }

  public int getStok(){
    return stok;
  }

  public void setStok(int stok){
    this.stok = stok;
  }
}

File: PHP/PetShop.php
<?php
class PetShop
{
  private $id = 0;
  private $nama = "";
  private $harga = 0;
  private $stok = 0;
  private $foto = ""; 

  // Konstruktor
  public function __construct($id, $nama, $harga, $stok, $foto)
  {
    $this->id = $id;
    $this->nama = $nama;
    $this->harga = $harga;
    $this->stok = $stok;
    $this->foto = $foto;
  }

  public function getId() { return $this->id; }
  public function getNama() { return $this->nama; }
  public function getHarga() { return $this->harga; }
  public function getStok() { return $this->stok; }
  public function getFoto() { return $this->foto; } 

  public function setId($id) { $this->id = $id; }
  public function setNama($nama) { $this->nama = $nama; }
  public function setStok($stok) { $this->stok = $stok; }
  public function setHarga($harga) { $this->harga = $harga; }
  public function setFoto($foto) { $this->foto = $foto; } 
}
?>
